{"uuid": "669f66f0-cd87-4076-858e-315d32ec2e0c", "children": ["de4f69ea-6ffa-45a5-9f99-ecf3cc2a4c52"], "befores": [{"name": "start_appium_desired", "status": "broken", "statusDetails": {"message": "NameError: name 'run' is not defined\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/zhangxiangkui/Downloads/AutomationTest/testcase/conftest.py\", line 22, in start_appium_desired\n    opt = eval(cmdopt)\n  File \"<string>\", line 1, in <module>\n"}, "start": 1587407157133, "stop": 1587407157133}], "start": 1587407157133, "stop": 1587407157183}